client/src/components/AdvisorReports/FilterableAdvisorReportsTable.js

```diff
@@ -140,9 +132,9 @@ class FilterableAdvisorReportsTable extends Component {
     return result
   }
 
-  downloadCSV(args) {
+  function downloadCSV(args) {
```

Comment:
Maybe it is nicer to have downloadCSV outside the component.

Reply:
Same here.